paket.dependencies

```diff
@@ -16,13 +20,14 @@ nuget Ionide.KeepAChangelog.Tasks copy_local:true
 nuget Expecto
 nuget Expecto.Diff
 nuget Expecto.TestResults
-nuget MedallionShell ~> 1.5.0
+nuget MedallionShell
```

Comment:
This has some weird transitive dependencies so I had to remove the constraint.  It looks kind of outdated so we may want to consider something else in the future.